
Showing logs for build [32mqbitflames-python-app-1-master-5vl26-3[0m stage [32mfrom-build-pack[0m and container [32mcredential-initializer[0m
{"level":"info","ts":1609684798.5891514,"caller":"creds-init/main.go:44","msg":"Credentials initialized."}

Showing logs for build [32mqbitflames-python-app-1-master-5vl26-3[0m stage [32mfrom-build-pack[0m and container [32mworking-dir-initializer[0m

Showing logs for build [32mqbitflames-python-app-1-master-5vl26-3[0m stage [32mfrom-build-pack[0m and container [32mplace-tools[0m

Showing logs for build [32mqbitflames-python-app-1-master-5vl26-3[0m stage [32mfrom-build-pack[0m and container [32mstep-git-source-qbitflames-python-app-1-master-5vl26-rvfmz[0m
{"level":"info","ts":1609684805.932218,"caller":"git/git.go:105","msg":"Successfully cloned https://github.com/qbitflames/python-app-1.git @ v0.0.3 in path /workspace/source"}
{"level":"warn","ts":1609684805.9323103,"caller":"git/git.go:152","msg":"Unexpected error: creating symlink: symlink /tekton/home/.ssh /root/.ssh: file exists"}
{"level":"info","ts":1609684805.9716048,"caller":"git/git.go:133","msg":"Successfully initialized and updated submodules in path /workspace/source"}

Showing logs for build [32mqbitflames-python-app-1-master-5vl26-3[0m stage [32mfrom-build-pack[0m and container [32mstep-setup-builder-home[0m

Showing logs for build [32mqbitflames-python-app-1-master-5vl26-3[0m stage [32mfrom-build-pack[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:
WARNING: no SHAs to merge, falling back to initial cloned commit

Showing logs for build [32mqbitflames-python-app-1-master-5vl26-3[0m stage [32mfrom-build-pack[0m and container [32mstep-setup-jx-git-credentials[0m
Generated Git credentials file /workspace/xdg_config/git/credentials

Showing logs for build [32mqbitflames-python-app-1-master-5vl26-3[0m stage [32mfrom-build-pack[0m and container [32mstep-build-python-unittest[0m

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK

Showing logs for build [32mqbitflames-python-app-1-master-5vl26-3[0m stage [32mfrom-build-pack[0m and container [32mstep-build-container-build[0m
E0103 14:40:08.683395      11 aws_credentials.go:77] while getting AWS credentials NoCredentialProviders: no valid providers in chain. Deprecated.
	For verbose messaging see aws.Config.CredentialsChainVerboseErrors
[36mINFO[0m[0000] Using dockerignore file: /workspace/source/.dockerignore 
[36mINFO[0m[0000] Retrieving image manifest python:onbuild     
E0103 14:40:08.926226      11 metadata.go:154] while reading 'google-dockercfg' metadata: http status code: 404 while fetching url http://metadata.google.internal./computeMetadata/v1/instance/attributes/google-dockercfg
E0103 14:40:08.928028      11 metadata.go:166] while reading 'google-dockercfg-url' metadata: http status code: 404 while fetching url http://metadata.google.internal./computeMetadata/v1/instance/attributes/google-dockercfg-url
[36mINFO[0m[0002] Retrieving image manifest python:onbuild     
[36mINFO[0m[0002] Built cross stage deps: map[]                
[36mINFO[0m[0002] Retrieving image manifest python:onbuild     
[36mINFO[0m[0003] Retrieving image manifest python:onbuild     
[36mINFO[0m[0004] Executing 3 build triggers                   
[36mINFO[0m[0004] Checking for cached layer gcr.io/weighty-nation-272311/cache:4603157fb951c8b4502529c64ddc571c85e381ce4686ea05728e14a56a118e66... 
[36mINFO[0m[0004] Using caching version of cmd: COPY requirements.txt /usr/src/app/ 
[36mINFO[0m[0004] Checking for cached layer gcr.io/weighty-nation-272311/cache:26208b704f381cfc1f456d1b1b9697de2e61930f2b836f1a4480488b7e191354... 
[36mINFO[0m[0005] Using caching version of cmd: RUN pip install --no-cache-dir -r requirements.txt 
[36mINFO[0m[0005] Checking for cached layer gcr.io/weighty-nation-272311/cache:09945693c50911caa3646651ba59b39930b7e02e88b06c18cd390f1e97c64bf1... 
[36mINFO[0m[0005] No cached layer found for cmd COPY . /usr/src/app 
[36mINFO[0m[0005] cmd: EXPOSE                                  
[36mINFO[0m[0005] Adding exposed port: 8080/tcp                
[36mINFO[0m[0005] Unpacking rootfs as cmd COPY . /usr/src/app requires it. 
[36mINFO[0m[0022] COPY requirements.txt /usr/src/app/          
[36mINFO[0m[0022] Found cached layer, extracting to filesystem 
[36mINFO[0m[0022] RUN pip install --no-cache-dir -r requirements.txt 
[36mINFO[0m[0022] Found cached layer, extracting to filesystem 
[36mINFO[0m[0023] COPY . /usr/src/app                          
[36mINFO[0m[0023] Resolving 114 paths                          
[36mINFO[0m[0023] Taking snapshot of files...                  
[36mINFO[0m[0023] Pushing layer gcr.io/weighty-nation-272311/cache:09945693c50911caa3646651ba59b39930b7e02e88b06c18cd390f1e97c64bf1 to cache now 
[36mINFO[0m[0023] COPY requirements.txt .                      
[36mINFO[0m[0023] Resolving 1 paths                            
[36mINFO[0m[0023] Taking snapshot of files...                  
[36mINFO[0m[0023] ENV PORT 8080                                
[36mINFO[0m[0023] No files changed in this command, skipping snapshotting. 
[36mINFO[0m[0023] EXPOSE 8080                                  
[36mINFO[0m[0023] cmd: EXPOSE                                  
[36mINFO[0m[0023] Adding exposed port: 8080/tcp                
[36mINFO[0m[0023] No files changed in this command, skipping snapshotting. 
[36mINFO[0m[0023] ENTRYPOINT ["python"]                        
[36mINFO[0m[0023] No files changed in this command, skipping snapshotting. 
[36mINFO[0m[0023] CMD ["app.py"]                               
[36mINFO[0m[0023] No files changed in this command, skipping snapshotting. 
[36mINFO[0m[0023] Pushing layer gcr.io/weighty-nation-272311/cache:bd6d00c2a5a37b083c59fba04ec78059c699f6ba1912267880231ce502442dd9 to cache now 

Showing logs for build [32mqbitflames-python-app-1-master-5vl26-3[0m stage [32mfrom-build-pack[0m and container [32mstep-build-post-build[0m
no CVE provider running in the current ** namespace so skip adding image to be analysed

Showing logs for build [32mqbitflames-python-app-1-master-5vl26-3[0m stage [32mfrom-build-pack[0m and container [32mstep-promote-changelog[0m
Converted /workspace/source/charts/python-app-1 to an unshallow repository
Generating change log from git ref 4bca399e32920e3cf41c1973539ef1be207e9de4 => 4679843dc1f4dd9c8b39a6653521869558df926e
WARNING: No release found for qbitflames/python-app-1 and tag v0.0.3 so creating a new release
Updated the release information at https://github.com/qbitflames/python-app-1/releases/tag/v0.0.3
generated: /workspace/source/charts/python-app-1/templates/release.yaml
Created Release python-app-1-0-0-3 resource in namespace **
Updating PipelineActivity qbitflames-python-app-1-master-3 with version 0.0.3
Updated PipelineActivities qbitflames-python-app-1-master-3 with release notes URL: https://github.com/qbitflames/python-app-1/releases/tag/v0.0.3

Showing logs for build [32mqbitflames-python-app-1-master-5vl26-3[0m stage [32mfrom-build-pack[0m and container [32mstep-promote-helm-release[0m
WARNING: No $CHART_REPOSITORY defined so using the default value of: http://jenkins-x-chartmuseum:8080
Adding missing Helm repo: storage.googleapis.com https://storage.googleapis.com/chartmuseum.jenkins-x.io
Successfully added Helm repository storage.googleapis.com.
Adding missing Helm repo: jenkins-x-chartmuseum http://jenkins-x-chartmuseum:8080
Successfully added Helm repository jenkins-x-chartmuseum.
WARNING: No $CHART_REPOSITORY defined so using the default value of: http://jenkins-x-chartmuseum:8080
Uploading chart file python-app-1-0.0.3.tgz to http://jenkins-x-chartmuseum:8080/api/charts
Received 401 response: {"error":"unauthorized"}
error: Failed to post chart to http://jenkins-x-chartmuseum:8080/api/charts due to response 401: {"error":"unauthorized"}
[31m
Pipeline failed on stage 'from-build-pack' : container 'step-promote-helm-release'. The execution of the pipeline has stopped.[0m
